packages/api/tests/expense.test.js

```diff
@@ -395,6 +395,7 @@ describe('Expense Tests', () => {
       const patchData = {
         value: expense.value + 5,
         note: 'patched in note',
+        expense_type_id: expense.expense_type_id,
```

Comment:
Instead of adjusting the tests, can we add a check to see if `expense_type_id` is included in the request body before verifying if it's deleted?